### DISCUSSION TITLE
add support for only showing Indeed Apply jobs

### DISCUSSION THREAD

**Discussion by JDBorges187** [damianmgarcia/Hide-n-Seek#9] [Ideas]
Hello, great job with the extension. I think it's a very valuable service as I never even considered hiding some jobs like confidential, promoted or spammy companies. I was wondering if you ever considered hiding jobs that weren't easily apply with linkedin or indeed easy apply? I do about 40-50 applications at a time with those and that would make the site much easier to navigate.

**Comment by damianmgarcia** [comment]
Hello @JDBorges187, and thank you for your suggestion!

## **Indeed**

Indeed has a problem that makes detection of "Easily apply" jobs a difficult task: **_they inconsistently apply the "Easily apply" label to their job listings_**.

The problem is exemplified in the screenshot below: There are 5 jobs. 3 of them have the "Easily apply" label (highlighted yellow). 2 of them do not have the "Easily apply" label ("Bus Detailer" and "Part-Time Team Member"). 

Based on this, one might intuit that jobs without the "Easily apply" label are not "Easily apply" jobs. However, that intuition would be incorrect: the 2 jobs without the "Easily apply" label are, in fact, "Easily apply" jobs. The "Bus Detailer" job, which does not have an "Easily apply" label is still an "Easily apply" job: it has the "Apply now" button (which only shows for "Easily apply" jobs).

**_That is the problem. Hide n' Seek depends on parsing the DOM for labels (class names or text), but if the labels are applied inconsistently by the website, Hide n' Seek cannot rely on them._**

![indeed-easy-apply-problem](https://user-images.githubusercontent.com/93714639/221716877-a9738af0-3ce5-4187-a791-e6dc1e8c6fbe.png)

The other option, opening every individual listing to detect the "Apply now" button is something that I do not see as feasible at this point.

## **LinkedIn**

Just in case any users are unaware: **LinkedIn has an "Easy Apply" filter**, so including one in Hide n' Seek would be redundant:

![linked-in-easy-apply](https://user-images.githubusercontent.com/93714639/221715076-e67506f1-95b2-46b2-aad4-30f2feaba82b.png)

## **Conclusion**

Thank you again for the suggestion. I will check Indeed periodically for other ways to detect "Easily apply" jobs, but for now, this feature cannot be added due to Indeed's inconsistent labeling.

**Comment by JDBorges187** [comment]
Oh wow! I didn’t even realize there were easy apply jobs without the label.
Super grateful for the write up. Hopefully some day they’ll expose it on
the front end some how with out needing to click the card.

**Comment by damianmgarcia** [comment]
Same here! Your suggestion prompted me to look into this, but previously I had always assumed that jobs without the "Easily apply" label were not "Easily apply" jobs.

Users who only want to apply to "Easily apply" jobs and who depend on the label may be missing out on jobs due to the inconsistent labeling.

I wrote Indeed support to inform them of this issue and I will update this thread if there is anything useful in the response.

**Comment by damianmgarcia** [comment]
From Indeed: "...the Product Team is aware of this issue, and they are currently running tests."
